content/04.results.md

```diff
@@ -0,0 +1,16 @@
+# Results
```

Comment:
One potential results panel: comparing results with abstract and full text (since you say above full text is not so often used).